src/current/cockroachcloud/connect-to-your-cluster.md

```diff
@@ -63,29 +63,7 @@ Azure Private Link is not yet available for [CockroachDB {{ site.data.products.d
 1. Run the command displayed on the **Accept VPC peering connection request** window using [Google Cloud Shell](https://cloud.google.com/shell) or using the [gcloud command-line tool](https://cloud.google.com/sdk/gcloud).
 1. On the **Networking** page, verify the connection status is **Available**.
 
-## Select a connection method
-
-1. In the top right corner of the Console, click the **Connect** button.
-
-    The **Connect** dialog displays with **IP Allowlist** selected by default.
-
-1.  Select a **Network Security** option:
-
-      You can use the **IP Allowlist** option if you have already [added an IP address to your allowlist.](#add-ip-addresses-to-the-allowlist)
-
-      For AWS clusters, you can select **AWS PrivateLink** if you have already [established a PrivateLink connection](#establish-gcp-vpc-peering-or-aws-privatelink).
-
-      For GCP clusters, you can select **VPC Peering** if you have already:
-    - [Enabled VPC peering while creating your cluster]({% link cockroachcloud/create-your-cluster.md %}#step-8-enable-vpc-peering-optional)
-    - [Established a VPC Peering connection](#establish-gcp-vpc-peering-or-aws-privatelink)
-
-1. From the **User** dropdown, select the SQL user you created.
-1. From the **Region** dropdown, select the region closest to where your client or application is running.
-1. From the **Database** dropdown, select the database you want to connect to.
-
-    The default database is `defaultdb`. For more information, see [Default databases](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/show-databases#preloaded-databases).
-
-1. Click **Next**.
```

Comment:
Why did we remove the above as part of this PR. It seems unrelated, but perhaps I'm missing something.

Reply:
I don't see anything to suggest that we're changing the experience for Dedicated users in this way here: https://cockroachlabs.atlassian.net/browse/CC-26935

Reply:
![Screenshot 2024-01-26 at 11 03 42 AM](https://github.com/cockroachdb/docs/assets/1447785/3fa02a9e-ad23-41c6-8714-089e3d72c013)
We seem to have dropped the instructions supporting the above ^^^

Reply:
This was my mistake -- I assumed that the Dedicated and Serverless modals were the same because I forgot about the recently-launched multi-step wizard. The Dedicated pages show where to select the database once again in recent commits.